erigon-lib/state/domain_test.go

```diff
@@ -2082,230 +2082,6 @@ func TestDomain_PruneProgress(t *testing.T) {
 	fmt.Printf("exitiig after %d iterations\n", i)
 }
 
-func TestDomain_Unwind(t *testing.T) {
```

Comment:
wait this test should pass either way with changesets, test doesnt rely on underlying knowledge